cypress/integration/pages/topicPage/tests.js

```diff
@@ -15,42 +15,36 @@ export default ({ service, pageType, variant }) => {
     beforeEach(() => {
       cy.log(Cypress.env('currentPath'));
       cy.log(service);
-      const env = Cypress.env('APP_ENV');
-      if (env !== 'local') {
-        // eslint-disable-next-line prefer-destructuring
-        topicId = Cypress.env('currentPath')
-          .split('topics/')
-          .pop()
-          .split('?')[0];
+      // eslint-disable-next-line prefer-destructuring
+      topicId = Cypress.env('currentPath').split('topics/').pop().split('?')[0];
 
-        if (scriptSwitchServices.includes(service)) {
-          if (service === 'serbian') {
-            otherVariant = variant === 'lat' ? 'cyr' : 'lat';
-          }
-          if (service === 'ukchina' || service === 'zhongwen') {
-            otherVariant = variant === 'simp' ? 'trad' : 'simp';
-          }
+      if (scriptSwitchServices.includes(service)) {
+        if (service === 'serbian') {
+          otherVariant = variant === 'lat' ? 'cyr' : 'lat';
+        }
+        if (service === 'ukchina' || service === 'zhongwen') {
+          otherVariant = variant === 'simp' ? 'trad' : 'simp';
         }
-
-        // Gets the topic page data for all the tests
-        cy.getPageData({
-          service,
-          pageType: 'topic',
-          variant,
-        }).then(({ body }) => {
-          topicTitle = body.data.title;
-          variantTopicId = body.data.variantTopicId;
-          pageCount = body.data.pageCount;
-          numberOfItems = body.data.curations[0].summaries.length;
-          firstItemHeadline = body.data.curations[0].summaries[0].title;
-          messageBanner = body.data.curations.find(
-            curation =>
-              curation.visualProminence === 'NORMAL' &&
-              curation.visualStyle === 'BANNER',
-          );
-        });
-        cy.log(`topic id ${topicId}`);
       }
+
+      // Gets the topic page data for all the tests
+      cy.getPageData({
+        service,
+        pageType: 'topic',
+        variant,
```

Comment:
No, because the topic id is retrieved in the getPageData custome command. In this file the topic ID is used further down the page in a different test https://github.com/bbc/simorgh/blob/b932969b5aa4c18b46d6a19c29de3cd66a874406/cypress/integration/pages/topicPage/tests.js#L268 .